Patches/EPOE Forked/Bionics_Patch.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+        <li>Expanded Prosthetics and Organ Engineering - Forked</li>
+    </mods>
+	<match Class="PatchOperationSequence">
+		<success></success>
+		<operations>
+		
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="EPOE_ScytherBlade"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
```

Comment:
I think it would be appropriate to add a Stab attack as well, since the Scyther mech has one as well.

Reply:
Added a stab attack and revised the cut attack.